node/cli/src/run_as_localchain.rs

```diff
@@ -1,15 +1,23 @@
 use crate::cli::Cli;
+use chrono::Local;
+use cli_opt::SealingMode;
 use sc_cli::SubstrateCli;
 
 pub fn run_as_localchain(cli: Cli) -> sc_service::Result<(), sc_cli::Error> {
+	// Temporary error message
+	if cli.sealing == SealingMode::Interval {
+		return Result::<(), _>::Err(sc_cli::Error::Input(String::from(
+			"Interval sealing is not yet implemented.",
+		)))
+	}
+
+	let current_time = Local::now();
+	let formatted_time = current_time.format("%Y-%m-%d %H:%M:%S").to_string();
+	println!("{} NO-RELAY MODE with {} sealing", formatted_time, cli.sealing);
```

Comment:
Don't we get the time part for free when we use the `log::info!` logging?

Reply:
Normally but info level logs are being filtered so it doesn't display.  Hence, the workaround.  If I could only modify the runner to append additional info.